taxi/forms.py

```diff
@@ -0,0 +1,69 @@
+from django import forms
+from django.core.exceptions import ValidationError
+from django.contrib.auth.forms import UserCreationForm
+from django.contrib.auth import get_user_model

```

Comment:
Good job on using `get_user_model()` as recommended in the checklist. However, it is not used in the rest of your code. Consider replacing `Driver` with `get_user_model()`.